src/Core/Geographic/Coordinates.js

```diff
@@ -106,6 +106,15 @@ function _convert(coordsIn, newCrs) {
                                    cartesian.x, cartesian.y, cartesian.z);
         }
 
+        if (coordsIn.crs === 'EPSG:4978' && newCrs === 'EPSG:4326') {
+            const geo = ellipsoid.cartesianToCartographic({
+                x: coordsIn._values[0],
+                y: coordsIn._values[1],
+                z: coordsIn._values[2],
+            });
+            return new Coordinates(newCrs, geo.latitude, geo.longitude, geo.h);
```

Comment:
* You have reversed longitude and latitude
* `cartesianToCartographic` return in Radian and EPSG:4326 is in degree